modules/scylladb/scylladb_test.go

```diff
@@ -0,0 +1,194 @@
+package scylladb_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"path/filepath"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	smithyendpoints "github.com/aws/smithy-go/endpoints"
+	"github.com/gocql/gocql"
+	"github.com/stretchr/testify/require"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/scylladb"
+)
+
+func TestScyllaDB(t *testing.T) {
+	ctx := context.Background()
+
+	ctr, err := scylladb.Run(ctx,
+		"scylladb/scylla:6.2",
+		scylladb.WithShardAwareness(),
+	)
+	testcontainers.CleanupContainer(t, ctr)
+	require.NoError(t, err)
+
+	t.Run("test without shard awareness", func(t *testing.T) {
```

Comment:
suggestion: we're standardising on simple names without spaces to ensure they match the output
```suggestion
	t.Run("without-shard-awareness", func(t *testing.T) {
```

More below